chore(doc): clarify vision / goal in README (#194)
Altinn/altinn-components: README.md

# altinn-components

> Altinn's frontend is developed by multiple product teams, all working toward the shared goal of delivering a seamless, unified experience—ensuring users perceive it as a single and robust product. 

Reviewer: ```suggestion
The new frontend of altinn.no is developed by multiple product teams, all working toward the shared goal of delivering a seamless, unified experience for the end users — ensuring they perceive it as a single and robust product. 
```
We should probably be specific about which frontend we are referring to: The frontend of altinn.no - the platform to be used by altinn's end users

This package represents a collaborative effort between designers and developers across these teams, consolidating solutions to shared challenges. 

> It provides Altinn-specific React components tailored to address the common UI needs of the various teams within Altinn.   

Reviewer: ```suggestion
It provides Altinn-specific React components tailored to address the common UI needs of these specific tems.   
```
We should probably be specific about the fact that this is for these teams so it's not misinterpreted as something to be used by any Altinn team, like Studio and Apps

Today these teams are [Team Arbeidsflate](https://github.com/digdir/dialogporten-frontend), [Team authorization](https://github.com/Altinn/altinn-access-management-frontend), and Team Portal.

> Note that this package is specifically designed to meet the unique needs of Altinn's product teams and is not intended to serve as a general-purpose library. 

Reviewer: ```suggestion
Note that this package is specifically designed to meet the unique needs of the aforementioned teams and is not intended to serve as a general-purpose library. 
```
"Altinn product teams" include teams like Apps, Studio, AppsMigration, etc. We should be weary of implying that this is inteded for them as well.
